docs/operate/reference/services/motion/_index.md

```diff
@@ -35,6 +35,128 @@ This defines the spatial context within which the motion service operates.
+
+{{< tabs >}}
+{{% tab name="From a client application" %}}
+
+To access the motion service built into `viam-server` from your client application code, use the resource name `builtin` to get a motion service client:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```python {class="line-numbers linkable-line-numbers"}
+# Get a motion service client
+motion_service = MotionClient.from_robot(machine, "builtin")
+
+# Then use the motion service, for example:
+moved = await motion_service.move(gripper_name, destination, world_state)
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```go {class="line-numbers linkable-line-numbers"}
+// Get a motion service client
+motionService, err := motion.FromRobot(machine, "builtin")
+if err != nil {
+  logger.Fatal(err)
+}
+
+// Then use the motion service, for example:
+moved, err := motionService.Move(context.Background(), motion.MoveReq{
+  ComponentName: gripperName,
+  Destination: destination,
+  WorldState: worldState
+})
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+If you created your own custom motion service, you can access it using the resource name you gave it in your machine's configuration.
+
+{{% /tab %}}
+{{% tab name="From within a module" %}}
+
+To access the motion service built into `viam-server` from your module code, you need to add the motion service as a [module dependency](/operate/get-started/other-hardware/create-module/dependencies/), using the resource name `builtin`.
+For example:
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+1. Edit your `validate_config` function to add the `builtin` motion service as a dependency so that it is available to your module.
+   You do not need to check for it in your config because the built-in motion service is always enabled.
+
+   ```python {class="line-numbers linkable-line-numbers"}
+   @classmethod
+   def validate_config(
+       cls, config: ComponentConfig
+   ) -> Tuple[Sequence[str], Sequence[str]]:
+       req_deps = []
+       req_deps.append("builtin")
+       return req_deps, []
+   ```
+
+1. Edit your `reconfigure` function to add the motion service as an instance variable so that you can use it in your module:
+
+   ```python {class="line-numbers linkable-line-numbers"}
+   def reconfigure(
+       self, config: ComponentConfig, dependencies: Mapping[ResourceName, ResourceBase]
+   ):
+       motion_resource = dependencies[Motion.get_resource_name("builtin")]
+       self.motion_service = cast(MotionClient, motion_resource)
+
+       return super().reconfigure(config, dependencies)
+   ```
+
+1. You can now use the motion service in your module, for example:
+
+   ```python {class="line-numbers linkable-line-numbers"}
+   def move_around_in_some_way(self):
+       moved = await self.motion_service.move(gripper_name, destination, world_state)
+       return moved
+   ```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+The following example assumes your module uses `AlwaysRebuild` and does not have a `Reconfigure` function defined.
```

Comment:
this makes me wonder what would change if it wasn't built that way - would it be difficult to cover that?